packages/renderer/src/lib/image/RenameImageModal.svelte

```diff
@@ -27,8 +27,8 @@ function disableSave(name: string, tag: string): boolean {
 }
 
 let imageNameErrorMessage = '';
-function validateImageName(event: any): void {
-  let inputName = event.target.value;
+function validateImageName(event: Event): void {
+  let inputName = (event.target as unknown as Input).value;
```

Comment:
issue: we cannot assume this cast

Reply:
Why ? Unless I miss something Input is where the event is coming from